### ISSUE TITLE
Cross-references acrosss multiple BibTeX files

### ISSUE THREAD

**Issue by salutis** [andras-simonyi/citeproc-el#87]
I have the following `.bib` file:

```
@Book{bennett+2015,
  editor       = {Tanya Long Bennett and Donna Gessell and Diana
                  Edelman-Young},
  title        = {Contribute a Verse},
  publisher    = {University of North Georgia Press},
  year         = 2015,
  subtitle     = {A Guide to First Year Composition},
  url          = {https://ung.edu/university-press/books/introduction-to-first-year-composition/}
}

@InBook{pearson-2015,
  author       = {J. Stephen Pearson},
  title        = {Writing in the Social and Physical Sciences},
  chapter      = 10,
  year         = 2015,
  pages        = {146-165},
  crossref     = {bennett+2015}
}
```

This works in Org and `citeproc.el`:

> Pearson, J. S. (2015). Writing in the social and physical sciences: A guide to first year composition (T. L. Bennett, D. Gessell, & D. Edelman-Young, Eds.; pp. 146–165). University of North Georgia Press. https://ung.edu/university-press/books/introduction-to-first-year-composition/

Alas, if I put the two BibTeX items into two separate files, it stops working, namely the rendered reference item has no `@Book` part:

> Pearson, J. S. (2015). Writing in the social and physical sciences (pp. 146–165).

I thought Org (or `citeproc.el`) concatenate all `.bib` files, and cross-references should thus work.

**Comment by andras-simonyi** [comment]
Thanks for reporting this, will try to push a fix shortly.

**Comment by andras-simonyi** [comment]
I've merged a PR (#88) which should fix this problem.

**Comment by salutis** [comment]
@andras-simonyi I can confirm that this works now. Thank you so much!

**Old:**

Pearson, J. S. (2015). _Writing in the social and physical sciences_ (pp. 146–165).

**New:**

Pearson, J. S. (2015). _Writing in the social and physical sciences: A guide to first year composition_ (T. L. Bennett, D. Gessell, & D. Edelman-Young, Eds.; pp. 146–165). University of North Georgia Press. https://ung.edu/university-press/books/introduction-to-first-year-composition/
